examples/common.h

```diff
@@ -63,6 +63,14 @@ extern int lg2_tag(git_repository *repo, int argc, char **argv);
  */
 extern void check_lg2(int error, const char *message, const char *extra);
 
+/**
+ * Read a file into a buffer
+ *
+ * @param path: path to the file that shall be read
```

Comment:
The build currently fails because the `:` is interpreted as part of the parameter name (`clang` tries to match the documentation with the signature). You should omit the `:`.

Reply:
Opps, fixed!